### PR TITLE
Rishi reporting for duty!

### PR DESCRIPTION
**Pull request by rishiosaur**

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -1,6 +1,16 @@
 {
   "docs_url": "https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md",
   "maintainers": [
+    {
+      "github_username": "rishiosaur",
+      "alumnus": true,
+      "show_on_website": true,
```
Comment (SleeplessByte) [inline config/maintainers.json:7]:
```suggestion
      "show_on_website": false,
```

File: config/maintainers.json [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+      "name": "Rishi Kothari",
+      "link_text": "Website",
+      "link_url": "https://rishi.js.org",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/15793261?s=460&u=dce39daca78b2f794644abdf061eed9a425d2a2c&v=4",
```
Comment (SleeplessByte) [inline config/maintainers.json:11]:
```suggestion
      "avatar_url": null,
```

We pull your GH avatar! So no need to set it :D

File: config/maintainers.json [hunk 3]
```diff
@@ -1,6 +1,16 @@
 {
   "docs_url": "https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md",
   "maintainers": [
+    {
```
Comment (SleeplessByte) [inline config/maintainers.json:4]: Can you also move yours to the bottom of the file <3 ?

Comment (rishiosaur) [reply config/maintainers.json:4]: Got it!

**Comment by SleeplessByte** [comment]
👋 Welcome! Just waiting for CI to pass ✅

File: config/maintainers.json [hunk 4]
```diff
@@ -100,6 +101,16 @@
...
+      "github_username": "rishiosaur",
+      "alumnus": true,
+      "show_on_website": false,
+      "name": "Rishi Kothari",
+      "link_text": "Website",
+      "link_url": "https://rishi.js.org",
+      "avatar_url": null,
+      "bio": "I program JS, Rust and Swift by day, and my own languages by night. 👋I'm Rishi, a JS developer with a passion for helping people! It's great to meet you :)"
+    },
```
Comment (SleeplessByte) [inline config/maintainers.json:114]:
```suggestion
    }
```

File: config/maintainers.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "docs_url": "https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md",
   "maintainers": [
+
```
Comment (SleeplessByte) [inline config/maintainers.json:4]:
```suggestion
```

Final change!

Comment (SleeplessByte) [reply config/maintainers.json:4]: (Remove the empty line)

**Comment by SleeplessByte** [comment]
Welcome to JavaScript!

**Pull request merged**